tests/testthat/test-gx-volcano.R

```diff
@@ -0,0 +1,20 @@
+#' Test for gx.volcanoPlot.XY
+#' 
+#' 
+
+#' Test for gx.volcanoPlot.LIMMA
+test_that("gx.volcanoPlot.LIMMA returns plot", {
```

Comment:
For most of the unit tests, I used hard data so that I could verify the output. However, I was not sure how to do the unit test for a plotting function. In this case, I just tested that the object was generated, but maybe there is some internals within it that I could use as reference.